### PR TITLE
civics.js and resource.js localization

### PR DESCRIPTION
**Pull request by Rodrigodd**
It only remains to finish space.js and vars.js

### CODE REVIEW

File: resources.js [hunk 1]
```diff
@@ -142,7 +143,7 @@ function loadResource(name,max,rate,tradable,stackable,color) {
     color = color || 'info';
     if (!global['resource'][name]){
         global['resource'][name] = {
-            name: name === 'Money' ? '$' : name,
+            name: name === races[global.race.species].name? name : name === 'Money' ? '$' : loc('resource_' + name + '_name'),
```
Comment (pmotschmann) [inline resources.js]: What's going on here with the races? race names aren't related to resource names.

File: resources.js [hunk 2]
```diff
@@ -153,7 +154,9 @@ function loadResource(name,max,rate,tradable,stackable,color) {
             rate: rate
         };
     }
-
+    else{
+        global['resource'][name].name = name === races[global.race.species].name? name : name === 'Money' ? '$' : loc('resource_' + name + '_name')
```
Comment (pmotschmann) [inline resources.js]: same issue as above

Comment (Rodrigodd) [reply resources.js]: One of the resources have the name of the race: 
line 104: loadResource(races[global.race.species].name,0,0,false,false,'warning');
So i need check if it is that resource before call loc('resource_' + name + '_name')

Comment (pmotschmann) [reply resources.js]: Ah ok

**Pull request merged**